lib/shared/image_viewer.dart

```diff
@@ -44,19 +45,20 @@ class _ImageViewerState extends State<ImageViewer> with TickerProviderStateMixin
   final GlobalKey<ScaffoldMessengerState> _imageViewer = GlobalKey<ScaffoldMessengerState>();
   final GlobalKey<ExtendedImageGestureState> gestureKey = GlobalKey<ExtendedImageGestureState>();
   bool downloaded = false;
-
   double slideTransparency = 0.92;
   double imageTransparency = 1.0;
-
   bool maybeSlideZooming = false;
   bool slideZooming = false;
   bool fullscreen = false;
   Offset downCoord = Offset.zero;
+  bool areImageDimensionsLoaded = false;
 
   /// User Settings
   bool isUserLoggedIn = false;
-
   bool isDownloadingMedia = false;
+  late double imagewidth = 0;
```

Comment:
For conventions, use camel case for the variable names (`imageWidth`, `imageHeight`, `maxZoomLevel`)

Reply:
@hjiangsu changes done